snippets/celsius_to_fahrenheit.md

```diff
@@ -0,0 +1,19 @@
+---
+title: celsius_to_fahrenheit
+tags: conversion, beginner
+---
+
+Converts Celsius to Fahrenheit.
+
+fahrenheit = (celsius * 1.8) + 32
+
+```py
+def celsius_to_fahrenheit(celsius):
+  return ((celsius * 1.8) + 32)
+```
+
+```py
+celsius_to_fahrenheit(25) # 77.0

```

Comment:
No need for that many examples, one will suffice as it's a straightforward use-case.